cypress/support/gutenberg/commands/controls.js

```diff
@@ -412,6 +415,100 @@ function radioControl( name, value, options = {} ) {
 		.click( { force: true } )
 }
 
+/**
+ * Command for adjusting the form token input.
+ *
+ * @param {string} name
+ * @param {*} value
+ * @param {Object} options
+ */
+function formTokenControl( name, value, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+		parentSelector,
+		supportedDelimiter = [],
+	} = options
+
+	const selector = () => cy.getBaseControl( name, {
+		isInPopover,
+		parentSelector,
+		supportedDelimiter,
+	} )
+
+	const isWpLessThan58 = compareVersions( Cypress.env( 'WORDPRESS_VERSION' ), '5.8.0', '<' )
+
+	beforeAdjust( name, value, options )
+	value.forEach( val => {
+		if ( isWpLessThan58 ) {
+		// The suggestions list does not display on WP versions less than 5.8. Type and enter the values.
+			selector()
+				.find( '.components-form-token-field__input' )
+				.click( { force: true } )
+				.type( val, { force: true } )
+		} else {
+			selector()
+				.find( '.components-form-token-field__input' )
+				.click( { force: true } )
+		}
+		selector()
+			.find( `ul.components-form-token-field__suggestions-list li:contains(${ val })` )
+			.click( { force: true } )
+	} )
+}
+
+/**
+ * Command for changing the style of the block.
+ *
+ * @param {string} value
+ */
+export function adjustBlockStyle( value = '' ) {
+	cy.get( '.block-editor-block-styles' )
+		.find( `div[aria-label="${ value }"]` )
+		.click( { force: true } )
+}
+
+/**
+ * Command for adjusting the checkbox control
+ *
+ * @param {string} name
+ * @param {boolean} value
+ * @param {Object} options
+ */
+function checkboxControl( name, value = true, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+		parentSelector,
+		supportedDelimiter = [],
+	} = options
+
+	const selector = name => cy
+		.getBaseControl( name, {
+			isInPopover, parentSelector, supportedDelimiter,
+		} )
+		.find( `.components-base-control__field:contains(${ name })` )
+
+	beforeAdjust( name, value, options )
+
+	selector( name ).find( 'svg.components-checkbox-control__checked' ).its( 'length' ).then( length => {
+		// Check if the checkbox is already true
+		if ( length > 0 ) {
+			// If the value is set to false, click the checkbox.
+			if ( ! value ) {
+				selector( name )
+					.find( 'input.components-checkbox-control__input' )
+					.click( { force: true } )
+			}
+		} else if ( value ) {
+			// If not yet checked and value is set to true, click the checkbox.
+			selector( name )
+				.find( 'input.components-checkbox-control__input' )
+				.click( { force: true } )
+		}
+	} )
```

Comment:
Comparing `length > 0` to value is cleaner. ( Assuming that `length > 0` = checkbox is checked )